### ISSUE TITLE
Start downloading from... selection unclear

### ISSUE THREAD

**Issue by shrx** [8cylinder/tv-overlord#57]
When adding a new show, we are prompted with this selection:

```
The last episode broadcast was S02E10.
Start downloading the [f]irst (or [enter]), [l]atest or season and episode? [f]:
```
It is unclear if it is possible to start downloading from an arbitrary season/episode and how to select the desired values.

**Comment by 8cylinder** [comment]
That is poorly worded, but I wasn't sure of a better way to say it.

`To specify a season and episode to start at, use two numbers.`
`eg: to start at S02E03, use: 2 3 instead of f or l.`
`Start downloading the [f]irst, [l]ast or season, episode? [f]:` 

This seems a bit wordy, but maybe this is the best way to say it?

**Comment by kwinczek** [comment]
I think there is slightly bigger problem there.

When adding new show, for which I have some episodes already downloaded and don't know next what episode is (if I have S1 E10 and don't know if next is S1 E11 or S2 E1) I have to say that I want to start downloading at S1 E10, but then tvol wants to download S1 E10 again, so I have to `editshow` and set S1 E10 to be last downloaded one and only then tvol will tell me what next one is..

So basically editing and adding shows uses different ways of specifying what is the last episode we have or want to download. Bit confusing. 

That plus original issue from above: I think best way of doing it would be to reword it to something like "What's the latest downloaded episode [ie.: (S)10 (E)22]?" for both `add` and `editshow`.

Also `editshow` could be shortened to `edit`.

While I am here - do you have a plan to provide zsh completion as well?

Awesome software, BTW, thanks!
